doc/dev/release-process.rst

```diff
@@ -244,7 +265,7 @@ Finally, when all appropriate testing and verification is done on the
 container images, run ``make-manifest-list.py --promote`` from the Ceph
 source tree (at ``container/make-manifest-list.py``) to promote them to
 their final release location on ``quay.io/ceph/ceph`` (you must ensure
-that you're logged into ``quay.io/ceph`` with appropriate permissions):
+that you're logged into ``quay.io/ceph`` and ``quay.ceph.io/ceph/prerelease`` with appropriate permissions):
```

Comment:
quay.io/ceph and quay.ceph.io/ceph are the repos, and the actual login points.  I've been fuzzy about that in the past but that's the way it works.  (prerelease is an 'image name' and :v19 is an 'image tag')